src/pages/Article/Article.tsx

```diff
@@ -3,25 +3,14 @@ import styled from 'styled-components';
 import mockArticle from './mockArticle';
 import ArticleContent from './ArticleContent/ArticleContent';
 import Comments from './Comments';
+import { ICommentItem } from './comment';
 
 const Container = styled.div`
   margin-left: 100px;
   margin-top: 110px;
   margin-right: 200px;
 `;
 
-interface CommentType {
-  author: {
-    id: number;
-    avatar: string;
-    username: string;
-  };
-  postTime: string;
-  like: number;
-  content: string;
-  childCommentList: CommentType[] | [];
-}
-
 interface ArticleContentType {
```

Comment:
可以把interface 放interfaces 文件夹去  以大写I 开头,  带type的 应该是type 不是interface

```suggestion
interface IArticleContent {
```